test/test_rosdep_duplicates_key.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2012, Willow Garage, Inc.
```

Comment:
Please update the year. And if you'd like contribute it to OSRF instead of Willow Garage.